keyvaluestore/managers.py

```diff
@@ -1,6 +1,7 @@
 from django.db import models
 from django.core.cache import cache
-from django.utils.translation import ugettext as _
```

Comment:
Why not also replace this with ugettext_lazy. Or does it behave weird when raising exception.

Reply:
Exceptions aren't really user-facing so it seems a bit weird to be translating those as well.